larpandoracontent/LArObjects/LArCaloHit.h

```diff
@@ -94,6 +104,19 @@ class LArCaloHit : public object_creation::CaloHit::Object
 class LArCaloHitFactory : public pandora::ObjectFactory<object_creation::CaloHit::Parameters, object_creation::CaloHit::Object>
 {
 public:
+    /**
+     *  @brief  Default constructor
+     *
+     */
+    LArCaloHitFactory();
+
+    /**
+     *  @brief  Constructor
+     *
+     *  @param  version the LArCaloHit version
+     */
+    LArCaloHitFactory(const unsigned int version);
```

Comment:
Is it worth having a `...(const unsigned int version = 1);` here, or is the default (explicitly zero argument) constructor required still?

Reply:
Agreed, I think the defaulted version is to be preferred here assuming there's no compiler shenanigans to worry about (I can't think of anything)